fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -302,12 +302,10 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
     private fun initTranslationsAction(context: Context, view: View) {
         val isEngineSupported =
             context.components.core.store.state.translationEngine.isEngineSupported
-        if (
-            !context.settings().enableTranslations &&
-            (
-                isEngineSupported == null ||
-                    isEngineSupported == false
-                )
+
+        if (isEngineSupported != true ||
+            !context.settings().enableTranslations ||
```

Comment:
Removing the secret settings `!context.settings().enableTranslations` in bug [1853058](https://bugzilla.mozilla.org/show_bug.cgi?id=1853058).